### PR TITLE
Email

### PR DESCRIPTION
**Pull request by liudmylasovetovs**
class email, email commands and validation

### CODE REVIEW

File: src/models/record.py [hunk 1]
```diff
@@ -7,12 +8,22 @@ class Record:
...
     def set_address(self, address):
         self.address = Address(address)
 
+    def add_email(self, email):
+        self.email = Email(email)
+
+    def remove_email(self):
+        self.email = None
+
+    def change_email(self, new_email):
```
Comment (Delrik) [inline src/models/record.py:25]: I suppose we can just remove it. It has the same logic as add_email

File: src/models/record.py [hunk 2]
```diff
@@ -7,12 +8,22 @@ class Record:
...
         self.phone = None
+        self.email = None
 
     def add_phone(self, phone):
         self.phone = Phone(phone)
 
     def set_address(self, address):
         self.address = Address(address)
 
+    def add_email(self, email):
```
Comment (Delrik) [inline src/models/record.py:19]: nitpick: Isn't it better to name it set_email?

Comment (liudmylasovetovs) [reply src/models/record.py]: renamed

File: src/models/record.py (hunk 1 above)
Comment (liudmylasovetovs) [reply src/models/record.py]: removed in next commit

**Review by Delrik — APPROVED** [review]

**Pull request merged**